### DISCUSSION TITLE
UV-K5/6 Calibration with multiple radios

### DISCUSSION THREAD

**Discussion by LessPaul** [armel/uv-k5-chirp-driver#41] [Q&A] [answered]
Each UV-K5/6 has unique calibration file programmed at the factory. So if I download from one UV-K5/6 and then clone to a second UV-J5/6, it would be my assumption that the calibration file on the second radio would be overwritten, and thus no longer optimal. 
Is this correct? 
If so, is there a work-around other than using k5prog to restore the factory calibration after doing the cloning?

**Comment by armel** [comment, marked as answer]
First of all, Hi.

No, Chirp does not replicate calibration data by default. You need to check a special option for this...
![chirp](https://github.com/user-attachments/assets/69d270ad-e014-4213-8674-e33a993acc10)

**Reply by LessPaul** [reply]
And Hello Armel! Thank you for clarifying my misunderstanding and answering my question.
